lib/network_utils.ml

```diff
@@ -1162,7 +1168,15 @@ module Ovs = struct
       in
       let vif_arg =
         let existing_vifs = List.filter (fun iface -> not (Sysfs.is_physical iface)) (bridge_to_interfaces name) in
-        List.flatten (List.map (fun vif -> ["--"; "--may-exist"; "add-port"; name; vif]) existing_vifs)
+        let ifaces_with_type =
+          let raw = vsctl ~log:true ["--bare"; "-f"; "table"; "--"; "--columns=name,type"; "find"; "interface"; "type!=\\\"\\\""] in
```

Comment:
If you want to avoid all the  extra quoting and slashes you can use `{|type!=\"\"|}` instead, it is equivalent to `"type!=\\\"\\\""`

Reply:
Interesting, learnt something new. Just update.

Reply:
It got introduced in OCaml 4.02: http://caml.inria.fr/pub/docs/manual-ocaml/manual036.html